src/EFTools/EntityDesign/VisualStudio/ModelWizard/ModelObjectItemWizard.cs

```diff
@@ -298,6 +302,20 @@ private static void PopluateReplacementDictionary(Project project, Dictionary<st
         [SuppressMessage("Microsoft.Maintainability", "CA1506:AvoidExcessiveClassCoupling")]
         public void RunFinished()
         {
+#if VS14ORNEWER
+            TelemetryService.DefaultSession.PostEvent(
+                new TelemetryEvent("vs/entityframework/entitydatamodelwizard")
```

Comment:
Not really sure what to use here. I tried to follow [the guidelines](https://wiki.vsdata.io/instrumentation_rules_naming_guidance).